### PR TITLE
Deprecation guide for with helper

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: content/ember/v3/with-helper.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+id: ember-glimmer.with-syntax
+title: Deprecate the with helper
+until: '4.0.0'
+since: 'Upcoming'
+---
+
+The use of `{{with}}` has been deprecated, you should replace it with either `{{let}}` or a combination of `{{let}}`, `{{if}}` and `{{else}}`:
```
Comment (Kennie-create) [inline content/ember/v3/with-helper.md:8]:
```suggestion
The use of `{{with}}` has been deprecated. You should replace it with either `{{let}}` or a combination of `{{let}}`, `{{if}}` and `{{else}}`: 
```

File: content/ember/v3/with-helper.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+id: ember-glimmer.with-syntax
+title: Deprecate the with helper
+until: '4.0.0'
+since: 'Upcoming'
```
Comment (Kennie-create) [inline content/ember/v3/with-helper.md:5]:
```suggestion
since: 'Upcoming Features'
```

**Comment by jenweber** [comment]
@locks Kennie and I paired this review together. Thanks for your work on it! We especially liked that there were 3 examples of different scenarios.

It needs to be "Upcoming Features" because "Upcoming" created a duplicate heading instead of grouping it with the others.

**Comment by locks** [comment]
Thank you for the swift review! I can't take credit for the content, I took it straight from the lovely RFC :)

**Review by jenweber — APPROVED** [review]

**Pull request merged**